crowbar_framework/config/initializers/filter_sql_log.rb

```diff
@@ -24,6 +24,8 @@ def sql(event)
         Rails.logger.debug(
           "SQL: INSERT/UPDATE transaction: filtering query to not expose potentially sensitive data"
         )
+      elsif event.payload[:sql] =~ /^(INSERT INTO|DELETE FROM|UPDATE) "delayed_jobs".*/
+        # Dont log SQL queries from delayed_jobs as they have their own log file
```

Comment:
Depending on the environment they might get logged to different places so I dont like to hardcode stuff in comments that can change in the future. Leads to incongruity in comments vs reality.